tests/letstest/scripts/test_apache2.sh

```diff
@@ -40,52 +59,54 @@ fi
 cd letsencrypt
 
 echo "Bootstrapping dependencies..."
-letsencrypt-auto-source/letsencrypt-auto --os-packages-only
+sudo tests/letstest/scripts/bootstrap_os_packages.sh
 if [ $? -ne 0 ] ; then
     exit 1
 fi
 
-# This script sets the environment variables PYTHON_NAME, VENV_PATH, and
-# VENV_SCRIPT based on the version of Python available on the system. For
-# instance, Fedora uses Python 3 and Python 2 is not installed.
-. tests/letstest/scripts/set_python_envvars.sh
-
-"$VENV_SCRIPT" -e acme[dev] -e certbot[dev,docs] -e certbot-apache
```

Comment:
Whoops. t looks like most of the rest of this script has been deleted or commented out.

Reply:
???????????????? oops